### DISCUSSION TITLE
"Permission denied (publickey)" only over specific network (SSH key configured)

### DISCUSSION THREAD

**Discussion by juliendargelos** [community/community#21755] [New to GitHub]
<p>I can’t clone/pull/push from github only over a specific network:</p>
<pre><code>git@github.com: Permission denied (publickey).
</code></pre>
<p>My SSH key is well configured on my machine and Github; I can clone/pull/push from my home’s network. I tested with other machines connected to the same network and they don’t have this problem.</p>
<p>This is the output of ssh -vT <a href="mailto:git@github.com">git@github.com</a>:</p>
<pre><code>OpenSSH_7.9p1, LibreSSL 2.7.3
debug1: Reading configuration data /etc/ssh/ssh_config
debug1: /etc/ssh/ssh_config line 48: Applying options for *
debug1: Connecting to github.com port 22.
debug1: Connection established.
debug1: identity file /Users/Julien/.ssh/id_rsa type 0
debug1: identity file /Users/Julien/.ssh/id_rsa-cert type -1
debug1: identity file /Users/Julien/.ssh/id_dsa type -1
debug1: identity file /Users/Julien/.ssh/id_dsa-cert type -1
debug1: identity file /Users/Julien/.ssh/id_ecdsa type -1
debug1: identity file /Users/Julien/.ssh/id_ecdsa-cert type -1
debug1: identity file /Users/Julien/.ssh/id_ed25519 type -1
debug1: identity file /Users/Julien/.ssh/id_ed25519-cert type -1
debug1: identity file /Users/Julien/.ssh/id_xmss type -1
debug1: identity file /Users/Julien/.ssh/id_xmss-cert type -1
debug1: Local version string SSH-2.0-OpenSSH_7.9
debug1: Remote protocol version 2.0, remote software version PaloAltoNetworks_0.2
debug1: no match: PaloAltoNetworks_0.2
debug1: Authenticating to github.com:22 as 'git'
debug1: SSH2_MSG_KEXINIT sent
debug1: SSH2_MSG_KEXINIT received
debug1: kex: algorithm: diffie-hellman-group-exchange-sha256
debug1: kex: host key algorithm: ssh-rsa
debug1: kex: server-&gt;client cipher: aes128-ctr MAC: hmac-sha1 compression: none
debug1: kex: client-&gt;server cipher: aes128-ctr MAC: hmac-sha1 compression: none
debug1: SSH2_MSG_KEX_DH_GEX_REQUEST(2048&lt;7680&lt;8192) sent
debug1: got SSH2_MSG_KEX_DH_GEX_GROUP
debug1: SSH2_MSG_KEX_DH_GEX_INIT sent
debug1: got SSH2_MSG_KEX_DH_GEX_REPLY
debug1: Server host key: **********
debug1: Host 'github.com' is known and matches the RSA host key.
debug1: Found key in /Users/Julien/.ssh/known_hosts:3
debug1: rekey after 4294967296 blocks
debug1: SSH2_MSG_NEWKEYS sent
debug1: expecting SSH2_MSG_NEWKEYS
debug1: SSH2_MSG_NEWKEYS received
debug1: rekey after 4294967296 blocks
debug1: Will attempt key: /Users/Julien/.ssh/id_rsa RSA ********** agent
debug1: Will attempt key: /Users/Julien/.ssh/id_dsa
debug1: Will attempt key: /Users/Julien/.ssh/id_ecdsa
debug1: Will attempt key: /Users/Julien/.ssh/id_ed25519
debug1: Will attempt key: /Users/Julien/.ssh/id_xmss
debug1: SSH2_MSG_SERVICE_ACCEPT received
debug1: Authentications that can continue: publickey
debug1: Next authentication method: publickey
debug1: Offering public key: /Users/Julien/.ssh/id_rsa RSA ********** agent
debug1: Server accepts key: /Users/Julien/.ssh/id_rsa RSA ********** agent
debug1: Authentications that can continue: publickey
debug1: Trying private key: /Users/Julien/.ssh/id_dsa
debug1: Trying private key: /Users/Julien/.ssh/id_ecdsa
debug1: Trying private key: /Users/Julien/.ssh/id_ed25519
debug1: Trying private key: /Users/Julien/.ssh/id_xmss
debug1: No more authentication methods to try.
git@github.com: Permission denied (publickey).
</code></pre>
<p>Any idea ?</p>
<p>Thanks for your help :slight_smile:</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>Thanks for reaching out.</p>
<p>For privacy and security reasons, we don’t have access to your private account details on the Community Forum. In order to troubleshoot this properly, you’ll need to reach out to private support at <a href="https://github.com/contact" rel="nofollow noopener">https://github.com/contact</a> or via email at <a href="mailto:support@github.com">support@github.com</a>.</p>
<p>Apologies that we can’t do more.</p>
